pkg/cmd/cli/cmd/status.go

```diff
@@ -160,7 +160,9 @@ func (o StatusOptions) RunStatus() error {
 		if err != nil {
 			return err
 		}
-		data, err := dot.Marshal(g, o.namespace, "", "  ", false)
+		// DOT graph names cannot contain dashes unless it is a double-quoted string
+		// See http://www.graphviz.org/doc/info/lang.html
+		data, err := dot.Marshal(g, fmt.Sprintf("%q", o.namespace), "", "  ", false)
```

Comment:
It is part of `github.com/gonum/graph` so we could make an upstream pull request to have `dot.Marshal` return an error when its passed an invalid name.